tests/unittests/graphOptzTest.cpp

```diff
@@ -2423,3 +2423,31 @@ TEST_F(GraphOptz, sinkTransposeBelowChannelShuffleNodesAndEliminate) {
   EXPECT_EQ(preShuffleRN->getDims(),
             llvm::makeArrayRef(expectedPreShuffleRNDims));
 }
+
+/// Test that convertPlaceholdersToConstants works properly with quantized
+/// types.
+TEST_F(GraphOptz, QuantizedFC) {
+  auto *input = mod_.createPlaceholder(ElemKind::Int8QTy, {2, 32}, 1.0, 0,
+                                       "input", false);
+  auto *weights = mod_.createPlaceholder(ElemKind::Int8QTy, {32, 32}, 1.0, 0,
+                                         "weights", false);
+  auto *bias =
+      mod_.createPlaceholder(ElemKind::Int32QTy, {32}, 1.0, 0, "bias", false);
+  auto *output = mod_.createPlaceholder(ElemKind::Int8QTy, {2, 32}, 1.0, 0,
+                                        "output", false);
+
+  auto *fc = F_->createFullyConnected("fc", input, weights, bias);
+  F_->createSave("save", fc, output);
+
+  ctx_.allocate(input);
+  ctx_.allocate(weights);
+  ctx_.allocate(bias);
+  ctx_.allocate(output);
+
+  glow::convertPlaceholdersToConstants(F_, ctx_, {input, output});
```

Comment:
I thought this stmt convert {input, output} to constant?

Reply:
It's actually the other way around, but now that you point it out, it does seem like a confusing API.  The reason it's designed that way is because in a lot of places we want to convert a ton of things (every weight) to constant, but only leave a couple things (I/O) as Placeholders :-/

Reply:
Sorry I read "!=" to "==" in the source code :P